addons/optics/XEH_preStart.sqf

```diff
@@ -1,11 +1,12 @@
 #include "script_component.hpp"
 
+if (configProperties [configFile >> "CBA_PIPItems"] isEqualTo [] && {
+    configProperties [configFile >> "CBA_CarryHandleTypes"] isEqualTo []
+}) exitWith {};
+
+#include "XEH_PREP.sqf"
+
 if (!hasInterface) exitWith {};
 
 // Do not move this below these checks, or this missing function can be exploited.
```

Comment:
looks like this comment became false with this commit

Reply:
Yes, sorry I am not done with this PR yet.